pkg/apiserver/registry/intelligence/throughputanomalydetector/rest.go

```diff
@@ -179,24 +245,68 @@ func (r *REST) Create(ctx context.Context, obj runtime.Object, createValidation
 
 func (r *REST) getTADetectorResult(id string, tad *v1alpha1.ThroughputAnomalyDetector) error {
 	var err error
+	query := tadQuery
+	switch tad.AggregatedFlow {
+	case "external":
+		query = aggTadExternalQuery
+	case "pod":
+		if tad.PodLabel != "" {
+			query = aggTadPodLabelQuery
+		} else if tad.PodName != "" {
+			query = aggTadPodNameQuery
+		}
```

Comment:
A similar situation here, if tad.PodLabel == "" and tad.PodName == "" then query = aggTadPodLabelQuery, I would suggest
```
               if tad.PodName != "" {
			query = aggTadPodNameQuery
		} else {
			query = aggTadPodLabelQuery
		}

```